### ISSUE TITLE
ux: keep track of where a user gets to in each guide

### ISSUE THREAD

**Issue by myitcv** [play-with-go/play-with-go#9]
Follow up to #8.

If a user accidentally or intentionally leaves a guide part way through, it is likely to be frustrating remembering where they got to.

Instead, we can track where a user got to and on reload of a page (or navigating back to a guide) we can take them directly to where they left off.

**Comment by myitcv** [comment]
cc @jboursiquot per our conversation.

**Comment by samschurter** [comment]
It would also be nice to see on the home page which guides have been completed/what percent is left, especially as more guides are added.

Love the guides so far, thanks for creating this!

**Comment by marcosnils** [comment]
> It would also be nice to see on the home page which guides have been completed/what percent is left, especially as more guides are added.

Good idea, well put it in the backlog to see what we can come up with. Preciate the feedback!
